### ISSUE TITLE
version 6.0.0 - trim() with too-large area specified causes black fill

### ISSUE THREAD

**Issue by jordanade** [BKWLD/croppa#206]
Version 5.1.0:
<img width="729" alt="Screen Shot 2022-05-17 at 11 23 58 PM" src="https://user-images.githubusercontent.com/2799445/168951310-cec4228d-2582-40af-9097-b77c3316686c.png">

Version 6.0.0:
<img width="739" alt="Screen Shot 2022-05-17 at 8 29 14 PM" src="https://user-images.githubusercontent.com/2799445/168951281-f0e349f1-b364-4c21-b5a1-50fb93312e27.png">

Here's the URL for that left-most image:
5wrrqhuMvUsSKnpXuDoIlWhbyyKDXK25CBS4kqJH-440x_-resize-trim(165,0,724,538).png

**Comment by sdebacker** [comment]
What do you mean by "on some images" ? I couldn't reproduce the problem with a JPEG file.

**Comment by jordanade** [comment]
I'll add more detail if I can figure it out. On first glance it seems to be affecting maybe 5-10%.

**Comment by jordanade** [comment]
It sort of looks like the resizing is happening BEFORE the trim...

**Comment by jordanade** [comment]
@sdebacker I figured it out. For whatever reason some stored crop specs in my system exceed the natural dimensions of the source image (when a source image was replaced with a smaller one, say). In previous versions this was apparently handled gracefully and the X and Y trim was limited to the natural X and Y size. This means quite a few of these crop specs went unnoticed and have accumulated in my database.

Can v6 emulate the previous behavior?
